### PR TITLE
Add Amazsell project

### PR DESCRIPTION
**Pull request by harryandriyan**
Added Amazsell to the projects of OSS Port.

### CODE REVIEW

File: projects/harryandriyan/amazsell.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+languages: # A list of programming languages used in this project
+  - JavaScript
+  - HTML / CSS
+  - React
+  - MongoDB
+  - express
+  - NodeJS
+  - Puppeteer
+tags: # A list of keywords relevant to the contributors for this project
+  - Scraping tool
```
Comment (ramonh) [inline projects/harryandriyan/amazsell.mdx:14]:
```suggestion
  - Scraping
```

Just noticed we already have the `Scraping` tag, so I'll go with this!

Comment (ramonh) [inline projects/harryandriyan/amazsell.mdx:14]:
```suggestion
  - Scraping
```

Thanks for the suggestion! We'll use the already existing `Scraping` tag here.

File: projects/harryandriyan/amazsell.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+name: amazsell.
+repoUrl: https://github.com/harryandriyan/amazsell
+description: Amazon Review Sucker & Classifier Tool!
+languages: # A list of programming languages used in this project
+  - JavaScript
+  - HTML / CSS
+  - React
+  - MongoDB
+  - express
```
Comment (ramonh) [inline projects/harryandriyan/amazsell.mdx:10]:
```suggestion
```

I think we can safely remove the `express` tag here, as it avoids bloating the list, and is covered by `NodeJS` :)

File: projects/harryandriyan/amazsell.mdx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  - NodeJS
+  - Puppeteer
```
Comment (ramonh) [inline projects/harryandriyan/amazsell.mdx:12]:
```suggestion
```

I think we can safely remove the `Puppeteer` tag here, as it avoids bloating the list :)

**Review by ramonh — CHANGES_REQUESTED** [review]
Thanks so much for your contribution @harryandriyan!

I made some suggestions, could you please take a look?

Thanks in advance!

**Comment by harryandriyan** [comment]
Hi @ramonh I've committed the changes, Thanks

**Review by ramonh — APPROVED** [review]
Looks great, thank you!

Welcome to Port 🚢

**Pull request merged**